### DISCUSSION TITLE
Ultimate guide for linting and  typescript

### DISCUSSION THREAD

**Discussion by donPuerto** [quasarframework/quasar#9083] [General - Components / Directives / etc]
Does anyone to share on how to work with typescript?
What are the plugins to install?
What are the settings to use?

**Comment by ontwikkelfabriek** [comment]
See here for Quasar V1:
https://quasar.dev/quasar-cli/supporting-ts#introduction

See here for Quasar V2(beta):
https://next.quasar.dev/quasar-cli/supporting-ts#introduction
